go/pkg/yolosvc/db.go

```diff
@@ -7,14 +7,16 @@ import (
 	"strings"
 	"time"
 
+	_ "github.com/mattn/go-sqlite3"
+
 	"berty.tech/yolo/v2/go/pkg/yolopb"
 	"github.com/jinzhu/gorm"
 	"github.com/mr-tron/base58"
 	"go.uber.org/zap"
 	"moul.io/zapgorm"
 )
 
-func initDB(db *gorm.DB, logger *zap.Logger) (*gorm.DB, error) {
+func InitDB(db *gorm.DB, logger *zap.Logger) (*gorm.DB, error) {
```

Comment:
this should not be exported

Reply:
@moul hmm testing.go (Init db is using this)
What we should do? eather i move back SetupDB from testing.go( in yolosvc) or this stays exported.

Reply:
isn't testing.go in the same package?

Reply:
It was you comment to move it to yolosvc @moul

Reply:
maybe it's related to the other PR #419 

but to keep things more simple, please, unexport it, since in the current implementation, you only call initDB from the same package (everything is in `yolosvc` in that PR)

and we'll see in the other PR how we manage things